frontend/app/src/components/settings/accounting/rule/AccountingRuleSetting.vue

```diff
@@ -360,159 +387,60 @@ const importFileDialog = ref<boolean>(false);
             </RuiButton>
             <AccountingRuleConflictsDialog
               v-if="conflictsDialogOpen"
-              :table-headers="cols"
               @close="conflictsDialogOpen = false"
               @refresh="refresh()"
             />
           </template>
-
-          <div class="w-full md:w-[25rem] ml-auto">
-            <TableFilter
-              :matches="filters"
-              :matchers="matchers"
-              @update:matches="updateFilter($event)"
-            />
+          <div class="flex flex-wrap gap-x-4 gap-y-2 items-center justify-between">
+            <RuiTabs
+              v-model="customRuleHandling"
+              color="primary"
+              class="border border-default rounded bg-white dark:bg-rui-grey-900 flex max-w-min"
+            >
+              <RuiTab :value="CUSTOM_RULE_HANDLING.EXCLUDE">
+                {{ t('accounting_settings.rule.tabs.regular') }}
+              </RuiTab>
+              <RuiTab :value="CUSTOM_RULE_HANDLING.ONLY">
+                {{ t('accounting_settings.rule.tabs.custom') }}
+              </RuiTab>
+            </RuiTabs>
+
+            <div class="w-full md:w-[25rem] ml-auto">
+              <TableFilter
+                :matches="filters"
+                :matchers="matchers"
+                @update:matches="updateFilter($event)"
+              />
+            </div>
           </div>
         </div>
       </template>
 
-      <CollectionHandler
```

Comment:
There shouldn't be too many instances of CollectionHandler, we should slowly phase them out completely.